EventFilter/L1TRawToDigi/plugins/implementations_stage2/BMTFUnpackerOutput_ZS.cc

```diff
@@ -27,14 +25,15 @@ namespace l1t
       res = static_cast<BMTFCollections*>(coll)->getBMTFMuons();
       //BxBlocks changed the format of the blocks
       int firstBX = 0, lastBX = 0;
-      for (auto bxBlock : bxBlocks) {
-      	int bx = bxBlock.header().getBx();
-      	if (bx < firstBX)
-      	  firstBX = bx;
+      int nBX = bxBlocks[0].header().getTotalBx();//how many BX included in the BxBlock before Suppression
```

Comment:
This will segfault if there are no BxBlocks in the vector. Can you check if the vector is not empty first?